app/controllers/replies_controller.rb

```diff
@@ -0,0 +1,54 @@
+class RepliesController < ApplicationController
+  REPLIES_PER_PAGE = 25
+  
+  before_action :require_logged_in_user_or_400
+  after_action :update_read_ribbons
+                
+  def show
+    @page = params[:page].to_i
+    if @page == 0
+      @page = 1
+    elsif @page < 0 || @page > (2 ** 32)
+      raise ActionController::RoutingError.new("page out of bounds")
```

Comment:
È normale che ci siano stringhe hardcodate in inglese? I18nizziamole, o al più hardcodiamole in italiano.

Reply:
`ActionController::RoutingError.new("page out of bounds")` visualizza una pagina d'errore con la scritta "page out of bounds", ma ovviamente solo in development mode. Lascerei così com'è.